src/App.tsx

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+let story: string[] = [];
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'null',
+  };
+
+  componentDidMount() {
+    const btn = document.getElementById('btn') as HTMLElement;
+
+    btn.addEventListener('click', () => {
+      story = [];
+    });
+
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+      story.push(event.key);
+    });
+  }
+
+  render() {
+    const { pressedKey } = this.state;
+
+    return (
+      <div className="App">
+        {story.length
+          ? (
+            <p className="App__message">
+              The last pressed key is
+              {' - '}
+              {`[${pressedKey}]`}
```

Comment:
```suggestion
              The last pressed key is
              {' - '}
              {`[${pressedKey}]`}
```
use interpolation here